devnet-sdk/system2/el.go

```diff
@@ -0,0 +1,47 @@
+package system2
+
+import (
+	"context"
+	"github.com/ethereum-optimism/optimism/op-service/client"
+	"github.com/ethereum-optimism/optimism/op-service/sources"
+	"github.com/ethereum/go-ethereum/core/types"
+)
+
+type EthClient interface {
+	SendTransaction(ctx context.Context, tx *types.Transaction) error
+
+	// TODO
+}
+
+type ELNode interface {
+	Common
+	EthClient
```

Comment:
I would rather have an accessor (`EthClient()`) - for both testing and composability purposes.

Reply:
We can do that